pkgs/by-name/sw/swaymux/package.nix

```diff
@@ -0,0 +1,40 @@
+{ cmake
+, fetchFromGitea
+, lib
+, nlohmann_json
+, qt6
+, qtbase
+, qtwayland
+, stdenv
+, wrapQtAppsHook
+}:
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.0";
+  pname = "swaymux";
+
+  src = fetchFromGitea {
+    rev = "v${finalAttrs.version}";
+    domain = "git.grimmauld.de";
+    owner = "Grimmauld";
+    repo = "swaymux";
+    hash = "sha256-M85pqfYnYeVPTZXKtjg/ks5LUl3u2onG9Nfn8Xs+BSA=";
+  };
+
+  buildInputs = [ qt6.qtwayland cmake nlohmann_json qtbase];
+  nativeBuildInputs = [ cmake qt6.wrapQtAppsHook ];
```

Comment:
Is `cmake` required in both ?

Reply:
no, and fixed. Also missed the fact qt6.qtbase should be used, as well as reduced the arguments requested.